recipe/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "laszip-python" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tmontaigu/laszip-python/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 31572639a928fabf7d60a340970c6ae256ae9382e0420e938d6ab237553ac57b
+  patches:
+    - patches/0001-Add-missing-includes.patch
+
+build:
+  number: 0
+  script:
+    - export CMAKE_GENERATOR="Ninja"  # [not win]
+    - set CMAKE_GENERATOR="Ninja"  # [win]
```

Comment:
```suggestion
    - set CMAKE_GENERATOR=Ninja  # [win]
```

Reply:
There is still a weird `FATALL_ERRORtoolchain:` string in the log.

It is coming from https://github.com/tmontaigu/laszip-python/blob/fe35a3b941ce1a55b01cd3951932520cd63a666d/cmake/FindLASzip.cmake#L4C1-L4C27

probably they meant to use `FATAL_ERROR` instead of `FATALL_ERROR` maybe to make it more visible